IS0402Test.py

```diff
@@ -1334,7 +1338,8 @@ def test_24_1(self, test):
             elif r.status_code == 200 and len(r.json()) > 0 or r.status_code != 200:
                 return test.OPTIONAL("Query API signalled that it does not support RQL queries. This may be "
                                      "important for scalability.",
-                                     "https://github.com/AMWA-TV/nmos/wiki/IS-04#registries-resource-query-language-rql")
+                                     "https://github.com/AMWA-TV/nmos/wiki/IS-04#registries-"
+                                     "resource-query-language-rql")
```

Comment:
In terms of being able to search for a URL of interest in the code-base, this is another place that breaking somewhere else might be better, e.g. before the fragment '#', if we are forced to break at all.

Or I had been wondering about introducing constants like `NMOS_WIKI_URL = "https://github.com/AMWA-TV/nmos/wiki"` (and similarly for the amwa-tv.github.io links) so we could write `NMOS_WIKI_URL + "/IS-04#registries-resource-query-language-rql"`?